src/components/Offers/Form/form-components/OfferForm.js

```diff
@@ -311,35 +337,39 @@ const OfferForm = ({ context, title }) => {
                                         />
                                     </Grid>
                                     <Grid item xs={12} className={classes.highlightOptionGrid}>
-                                        <MultiOptionTextField
-                                            values={contacts}
-                                            label="Contacts *"
-                                            itemLabelPrefix="Contact #"
-                                            controllerName="contacts"
-                                            onAdd={appendContact}
-                                            onRemove={removeContact}
-                                            getValues={getValues}
-                                            control={control}
-                                            errors={errors.contacts || requestErrors.contacts}
-                                            disabled={formDisabled}
-                                            addEntryBtnTestId="contacts-selector"
-                                        />
+                                        <div id="contacts">
```

Comment:
What are the new divs (contacts and requirements) used for? i didn't quite get it

Reply:
I needed to place an id for each, but the `MultiOptionTextField` doesn't let me do it as it is. I chose to do this instead of modifying `MultiOptionTextField`, since it is simpler and it doesn't require renaming a bunch of things (more specifically, the component uses ids for each option and has a variable named `id` in the middle of the confusion).

Reply:
I removed the divs nonetheless.